src/database/mod.rs

```diff
@@ -434,6 +435,65 @@ impl Database {
         Ok(identity)
     }
 
+    // TODO: Change status to enum
+    pub async fn get_unprocessed_commitments(
+        &self,
+        status: Status,
+    ) -> Result<Vec<types::UnprocessedCommitment>, Error> {
+        let query = sqlx::query(
+            r#"
+                SELECT * FROM unprocessed_identities
+                WHERE status = $1
+            "#,
+        )
+        .bind(<&str>::from(status));
+
+        let result = self.pool.fetch_all(query).await?;
+
+        Ok(result
+            .into_iter()
+            .map(|row| types::UnprocessedCommitment {
+                commitment:    row.get::<Hash, _>(0),
+                status:        row.get::<&str, _>(1).parse().expect("status is unreadable"),
```

Comment:
We should probably extend the FromStr trait for Status and use it here
https://github.com/worldcoin/signup-sequencer/blob/f9790a115e6a7fea2a746f567b3374bbc656364a/src/identity_tree.rs#L58